taxonomy/emsi_client.py

```diff
@@ -0,0 +1,241 @@
+
```

Comment:
This file was added by @hasnain-naveed in https://github.com/edx/taxonomy-service/pull/2 
You would need to rebase and make sure there are no duplications.